governance/xc_admin/packages/xc_admin_frontend/components/InstructionViews/WormholeInstructionView.tsx

```diff
@@ -408,12 +409,29 @@ export const WormholeInstructionView = ({
                 <CopyText text={'0x' + governanceAction.callAddress} />
               </div>
               <div>Value: {governanceAction.value.toString()}</div>
-              <div>
-                Call Data:{' '}
-                <CopyText
-                  text={'0x' + governanceAction.calldata.toString('hex')}
-                />
-              </div>
+              {(() => {
+                const parsedData = parseEvmExecuteCallData(
+                  governanceAction.calldata.toString('hex')
+                )
+                if (parsedData === undefined) return <></>
```

Comment:
any rationale behind this?

Reply:
I think the most important part of this ui elements is verification. Not showing unparsable data can be confused with not having any sort of data. We should tell the user that there was some data that we can not parse and do not know what it actually is so that they become extra careful and verify it in some other way

Reply:
oh yeah, for that on the UI i am showing the call data. 
it is just for this method that I have returned undefined if we are not able to parse.

Reply:
I made the change to show the calldata UI after your comment. IMO it is more intuitive to have this function return undefined if it can't parse.